### PR TITLE
Update m1s preset

### PR DESCRIPTION
**Pull request by Garume**
Update m1s presets
- Added M1S RainingCats
- Improve M1S Double/Quadruple Swipe
    - Added Text (4 people / Pair)
    - Added Trriger Event (support to clone cast)
    - Change color red → green

### CODE REVIEW

File: Presets/Dawntrail/Raids/The Arcadion 1 Savage.md [hunk 1]
```diff
@@ -110,3 +110,9 @@ This displays red when a tile will receive damage and black when a tile will bre
...
...lColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38032],"refActorUseCastTime":true,"refActorCastTimeMax":17.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line5","type":3,"refY":20.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[38034],"refActorUseCastTime":true,"refActorCastTimeMin":15.0,"refActorCastTimeMax":18.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0},{"Name":"Line6","type":3,"refY":33.0,"radius":3.0,"color":4278190335,"fillIntensity":0.3,"originFillColor":838861055,"endFillColor":838861055,"refActorNPCNameID":13071,"refActorRequireCast":true,"refActorCastId":[39632],"refActorUseCastTime":true,"refActorCastTimeMin":16.0,"refActorCastTimeMax":19.0,"refActorUseOvercast":true,"refActorComparisonType":6,"includeRotation":true,"refActorTetherTimeMin":0.0,"refActorTetherTimeMax":0.0,"FillStep":2.0}]}
```
Comment (Aimsucks) [inline Presets/Dawntrail/Raids/The Arcadion 1 Savage.md:117]: I would update the name to have a space between `Raining` and ` Cats`:
```json
"Name":"M1S Raining Cats"
```

**Comment by Aimsucks** [comment]
Everything else looks good besides the name of the layout!

File: Presets/Dawntrail/Raids/The Arcadion 1 Savage.md (hunk 1 above)
Comment (Garume) [reply Presets/Dawntrail/Raids/The Arcadion 1 Savage.md]: Fixed.
Thank you.

**Comment by Limiana** [comment]
Thank you!

**Pull request merged**